src/Workspaces/Core/Portable/Remote/RemoteSupportedLanguages.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    internal static class RemoteSupportedLanguages
```

Comment:
I dont know what is better name. but without Remote, kind of conflict with other general ones. I will leave it like this for now.